packages/govern-server/src/env.ts

```diff
@@ -1,4 +1,14 @@
-export function graphqlHttpPort(): number {
+function graphqlHttpPort(): number {
```

Comment:
Instead of using environment variables could we do it with configuration objects cause with it are those helpers wouldn't be required.

Reply:
I like being able to configure the server without touching any file. Or do you see a way to have both?

Reply:
Not sure if actually was this tool I thought to use for web3.js but it was definitely something similar: https://github.com/commenthol/configg

... with such config management would you just have one environment variable you set.. for example ``production`` or ``local``

Reply:
... but we can change this later after ``Phase 4``